src/components/field-types/RadioFieldComponent.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <validate :state="fieldState" :custom="{'validate': isValid, 'unique': isUnique}" v-if="options.length > 0">
+  <validate :state="fieldState" :custom="{'validate': isValid, 'unique': isUnique}" v-if="options.length > 0" :debounce="1">
```

Comment:
maybe add a note about the magic '1' , on why it is there or use a named constant